concepticondata/conceptrelations.tsv

```diff
@@ -1217,3 +1217,19 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3074	FIX	instanceof	2243	REPAIR
 1755	UNDRESS	instanceof	3538	TAKE OFF
 158	PUT ON	antonymof	3538	TAKE OFF
+3603	EARTH OR LAND	instanceof	1228	LAND
+3603	EARTH OR LAND	instanceof	626	LAND
+3604	LIGHT BLUE	instanceof	837	BLUE
+3604	LIGHT BLUE	narrower	837	BLUE
+3605	LIGHT GREEN	instanceof	1425	GREEN
+3605	LIGHT GREEN	narrower	1425	GREEN
```

Comment:
I would rather say: these are distinct colors, so we should not subsume them, since there are external classifications that do a better job here (otherwise, we'd also have to subsume yellow and orange under BROWN, etc.).

Reply:
So just delete the relations.